python-package/xgboost/sklearn.py

```diff
@@ -717,15 +771,17 @@ def predict(self, data, output_margin=False, ntree_limit=None,
         .. note:: This function is not thread safe.
 
           For each booster object, predict can only be called from one thread.
-          If you want to run prediction using multiple thread, call ``xgb.copy()`` to make copies
-          of model object and then call ``predict()``.
+          If you want to run prediction using multiple thread, call
+          ``xgb.copy()`` to make copies of model object and then call
+          ``predict()``.
 
         .. note:: Using ``predict()`` with DART booster
 
-          If the booster object is DART type, ``predict()`` will perform dropouts, i.e. only
-          some of the trees will be evaluated. This will produce incorrect results if ``data`` is
-          not the training data. To obtain correct results on test sets, set ``ntree_limit`` to
-          a nonzero value, e.g.
+          If the booster object is DART type, ``predict()`` will perform
+          dropouts, i.e. only some of the trees will be evaluated. This will
+          produce incorrect results if ``data`` is not the training data. To
+          obtain correct results on test sets, set ``ntree_limit`` to a nonzero
+          value, e.g.
 
           .. code-block:: python
 
```

Comment:
Side comment: this note is no longer necessary, since `predict()` will no longer drop trees by default. See #5115 and #5204. Also need to remove the same note from `XGBModel.predict()` too.

Reply:
Done.